Cyb3rWard0g/HELK
# New import script + saved objects

**What is this PR for?**
This PR is for full support of Kibana 7 and contains:
- the dashboards converted to Kibana 7
- a new way of importing saved objects compliant with Kibana 7
- a script to export all current objects from Kibana

**What type of PR is it?**
[Update]

**How should this be tested?**
Try to make a new install with this branch and see if everything gets imported properly.

**Questions:**
* Do the licenses files need update? No
* Are there breaking changes for older versions? Not sure
* Does this needs documentation? No
